src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -309,6 +313,32 @@ const formConfig = {
           uiSchema: prisonerOfWar.uiSchema,
           schema: prisonerOfWar.schema,
         },
+        // Ancillary forms wizard
```

Comment:
You don't say 😛

Reply:
Yeah, I thought it might be helpful to group pages like this sort of unofficially. :man_shrugging: